tests/unit/CurrencyUtilsTest.ts

```diff
@@ -113,7 +113,23 @@ describe('CurrencyUtils', () => {
             [2500, 25],
             [2500.5, 25], // The backend should never send a decimal .5 value
         ])('Correctly converts %s to amount in units handled in frontend as an integer', (amount, expectedResult) => {
-            expect(CurrencyUtils.convertToFrontendAmountAsInteger(amount)).toBe(expectedResult);
+            expect(CurrencyUtils.convertToFrontendAmountAsInteger(amount, CONST.CURRENCY.USD)).toBe(expectedResult);
+        });
+    });
+
+    /**
+     * VND uses 0 decimals, so this test is needed.
+     * https://github.com/Expensify/App/pull/43948
+     */
+    describe('convertToFrontendAmountAsInteger VND', () => {
```

Comment:
https://github.com/Expensify/App/blob/8d6b1b3711e95421cdfe145f0976b0b37f0ee19a/tests/unit/CurrencyUtilsTest.ts#L134-L144

I think we should do like above reference instead of creating new test